core/src/main/java/org/apache/iceberg/rest/requests/ReportMetricsRequestParser.java

```diff
@@ -61,7 +61,11 @@ private static String fromReportType(ReportType reportType) {
   }
 
   private static ReportType toReportType(String type) {
-    return ReportType.valueOf(type.replaceAll("-", "_").toUpperCase(Locale.ROOT));
+    try {
```

Comment:
Here as well, we typically put these methods in the enum class so people can easily find them.